### PR TITLE
Update Travis config

### PR DESCRIPTION
**Pull request by bonniegee**
Updates Travis config as part of CI maintenance. I was getting a bit tripped up by using `PORT` as a variable, since that variable is also used in deployment. Do I not need to worry about it since it will be overwritten?

The README will need to be updated, but I'll do that in a separate PR as I want to move this forward before finishing setting up deployment on production (and making these changes in pictor).

I know there is an open dependency PR but I decided not to include that fix here, since the most recent version of Django is still using sqlparse `0.4.1`.

### CODE REVIEW

**Review by helrond — CHANGES_REQUESTED** [review]
Cool this looks good! I like the addition of the APPLICATION_NAME variable, and am going to roll that into my changes to Zodiac.

Regarding the port - the environment variables you set in Travis CI are available here, so you could just target $DJANGO_PORT. Again, I'll update that in Zodiac.

Can you add an `after_script:` section to print out the coverage report? That helps us measure how good unit test coverage is. See [zodiac](https://github.com/RockefellerArchiveCenter/zodiac/blob/ci/.travis.yml#L18-L19)

Last, it is possible to update sqlparse - you'll need to delete the requirements.txt file and run `pip-compile`.

**Comment by bonniegee** [comment]
> Regarding the port - the environment variables you set in Travis CI are available here, so you could just target $DJANGO_PORT. Again, I'll update that in Zodiac.

In the case of $DJANGO_PORT, we have that value as specific to each branch for this application, which would mean we would need to set it in this file for Travis to successfully run on other branches?

**Comment by helrond** [comment]
> In the case of $DJANGO_PORT, we have that value as specific to each branch for this application, which would mean we would need to set it in this file for Travis to successfully run on other branches?

Yes, that's right - good point. I wonder if there's a way for us to more reliably tie all of the ports together - there's stuff in configs but also the entrypoint uses a port number that is hardcoded. And maybe other places as well?

File: docker-compose.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ services:
     volumes:
       - .:/code
     ports:
-      - "8000:8000"
+      - "8009:8000"
```
Comment (helrond) [inline docker-compose.yml]:
```suggestion
      - "8009:8009"
```

File: .travis.yml [hunk 2]
```diff
@@ -1,27 +1,23 @@
 language: python
+cache: pip
 services:
   - docker
 env:
   global:
-    - DOCKER_COMPOSE_VERSION: 1.25.5
     - APPLICATION_NAME: aquila
     - SERVICE_NAME: aquila-web
```
Comment (helrond) [inline .travis.yml]: Two things: 
- we've called the `SERVICE_NAME` variable `CONTAINER` in other contexts and I'd prefer to be consistent. 
- I think it makes sense to use a `DJANGO_PORT` variable here, even though there's also a environment variable already defined in Travis for deployment. Getting these ports all aligned as variables is going to take some more thinking, but in the meantime it's useful to have that pulled out as a variable so we remember we have to deal with it.

File: .travis.yml [hunk 3]
```diff
@@ -1,27 +1,23 @@
...
-  - sudo mv docker-compose /usr/local/bin
   - cp ${APPLICATION_NAME}/config.py.example ${APPLICATION_NAME}/config.py
-  - echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
-  - docker-compose build
+  - echo "${DOCKER_PASSWORD}" | docker login -u "${DOCKER_USERNAME}" --password-stdin
   - docker-compose up -d
-  - docker-compose exec ${SERVICE_NAME} pip install pre-commit
-  - docker-compose exec ${SERVICE_NAME} pip install coverage
+install:
+  - ./wait-for-it.sh ${SERVICE_NAME}:8009 -- docker-compose exec ${SERVICE_NAME} pip install coverage pre-commit
```
Comment (helrond) [inline .travis.yml]: Use a variable for the port number as described above.

File: .travis.yml [hunk 4]
```diff
@@ -33,7 +29,8 @@ deploy:
     skip_cleanup: true
     on: &development
```
Comment (helrond) [inline .travis.yml]:
```suggestion
    on:
```

**Review by helrond — CHANGES_REQUESTED** [review]
A couple of minor things.

**Review by helrond — APPROVED** [review]

**Review by HaSistrunk — APPROVED** [review]

**Review by p-galligan — APPROVED** [review]

**Pull request merged**